test/parallel/test-runner-watch-mode-complex.mjs

```diff
@@ -46,8 +50,12 @@ test('test to delete has ran');`,
 const fixturePaths = Object.fromEntries(Object.keys(fixtureContent)
   .map((file) => [file, tmpdir.resolve(file)]));
 
-Object.entries(fixtureContent)
-  .forEach(([file, content]) => writeFileSync(fixturePaths[file], content));
+async function setupFixtures() {
+  await Promise.all(Object.entries(fixtureContent)
+    .map(([file, content]) => writeFile(fixturePaths[file], content)));
+}
+
+await setupFixtures();
```

Comment:
Do you need to await both `Promise.all()` and `setupFixtures()`?

Reply:
done (my bad 😬 )